calc/calculator.js

```diff
@@ -0,0 +1,19 @@
+class Calc {
+  constructor() {
+}
+
+sum(a, b) {
+    let firstNumber = +a;
+    let secondNumber = +b;
+    if (isNaN(firstNumber) || isNaN(secondNumber)) {
+        return NaN;
+    }
+    return firstNumber + secondNumber;
+}
+};
+const calculator = new Calc();
+
+export default calculator;
```

Comment:
Not sure this is a good idea to create a class instance inside **Calc.js** file. You deprive other developers ability to create new objects or inherits from this.